### DISCUSSION TITLE
Questions about saving the game

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1255] [Question and answer] [answered]
Hi, I'm quite new to dedicated servers and headless clients. I recently started hosting a server with TADST on my old laptop. I also set up a headless client on the same hosting computer. However, I don't understand where the save files are located, if I lose the saves closing the server, and if I need to set up the server with the "Persistent Battlefield" option. I would appreciate it if someone can answer my questions! **Thanks in advance!**

**Comment by Vdauphin** [comment, marked as answer]
Hello,
https://github.com/Vdauphin/HeartsAndMinds/issues/449#issuecomment-352271515
Cheers

**Reply by Vsauce07** [reply]
Thank you!
